pkg/server/options.go

```diff
@@ -56,21 +55,6 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.AddGoFlagSet(flag.CommandLine)
 }
 
-// Validate will check command line options
```

Comment:
How about to keep Validate() function as check runtime kind (docker/cri)? Otherwise the program accepts other runtime (kataContainer?) even though we don't support it.

Reply:
So validation is done now in the [Set](https://github.com/k8snetworkplumbingwg/multi-networkpolicy-iptables/blob/a276201e9ca8359c40a08c8c617bef3f3e683830/pkg/controllers/pod.go#L52-L60) method of the [Value](https://golang.org/pkg/flag/#Value) interface.

I would keep validate if we had to do cross-argument validation but I haven't found the use for it right now.

Reply:
I'm OK with this since we're still keeping the validation.